src/lib/getDataFromExternalSources.test.ts

```diff
@@ -52,4 +52,134 @@ describe('getDataFromExternalSources', () => {
       );
     }).to.not.throw();
   });
+
+  it("should return the right data when the schema's valueContent is VerifiableURI", async () => {
+    const schema: ERC725JSONSchema = {
+      name: 'LSP3Profile',
+      key: '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+      keyType: 'Singleton',
+      valueType: 'bytes',
+      valueContent: 'VerifiableURI',
+    };
+
+    const dataFromChain: DecodeDataOutput[] = [
+      {
+        name: 'LSP3Profile',
+        key: '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+        value: {
+          verification: {
+            data: '0xdb864ed42104cee179785036cb4ff1183ebc57e5532ae766ad8533fa48acfbb3',
+            method: 'keccak256(utf8)',
+          },
+          url: 'ipfs://QmdMGUxuQsm1U9Qs8oJSn5PfY4B1apGG75YBRxQPybtRVm',
+        },
+      },
+    ];
+
+    const result = await getDataFromExternalSources(
+      [schema],
+      dataFromChain,
+      'https://api.universalprofile.cloud/ipfs/',
```

Comment:
Please mock this as we can't rely on outside API/server for the testing of this lib.